lib/python/script/core.py

```diff
@@ -129,6 +129,8 @@ def scan(gisbase, directory):
         dir_path = os.path.join(gisbase, directory)
         if os.path.exists(dir_path):
             for fname in os.listdir(os.path.join(gisbase, directory)):
+                if fname == '__pycache__':
```

Comment:
This change is probably not Cmake related and should be separated out

Reply:
It seems that there is no __pycache__ in $GISBASE/bin and $GISBASE/scripts directories.